src/pages/ammo/index.js

```diff
@@ -14,53 +14,21 @@ import CenterCell from '../../components/center-cell';
 import ValueCell from '../../components/value-cell';
 import TraderPriceCell from '../../components/trader-price-cell';
 import { useItemsQuery } from '../../features/items/queries';
-import rawData from '../../data/ammo.json';
+import { formatCaliber } from '../../modules/format-ammo';
+import symbols from '../../symbols';
 
 import './index.css';
 
 const MAX_DAMAGE = 170;
 const MAX_PENETRATION = 70;
```

Comment:
Do we have any context around why these values are hard coded for max damage and max penetration? Is this a BSG thing or something we are doing for some reason?

Reply:
My understanding is that those are the max values to prevent outliers from zooming the graph out too far and to keep the armor pen levels accurate on the y axis.